SharedSources/MediaLibraryService.swift

```diff
@@ -361,7 +361,7 @@ extension MediaLibraryService {
 extension MediaLibraryService {
     func requestThumbnail(for media: [VLCMLMedia]) {
         media.forEach() {
-            guard !$0.isThumbnailGenerated() else { return }
+            if let thumbnail = $0.thumbnail(of: .thumbnail), !thumbnail.path.isEmpty { return }
```

Comment:
`isThumbnailGenerated` should do the trick here since we always use `.thumbnail`.

Additionally, with the next release of the wrapper, nil will be returned on empty MRLs.